README.md

```diff
@@ -45,7 +45,14 @@ If the repository has multiple deployables, like a database and app service, sep
        - v1, v1.0, v1.0.0, v1.0.0-test1
        - 2, 2.0, 2.0.0, 2.0.0-test2
   - Branch: Default when the SHA or Tag pattern do not match
-
+- Workflows with `success` or `failure` deploy status
+  - An issue will be created in the appropriate environment column on the project board if one does not exist.
+  - If there is an existing issue representing this deploy, it will be moved to the appropriate environment column on the project board.
+  - The `🚀currently-in-<env>` label will be removed from the issue that currently has it (if there is one) and put the label on the issue representing this deploy.
+- Workflows with `cancelled` or `skipped` deploy status
+  - An issue will be created in the appropriate environment column on the project board if one does not exist.
+  - If there is an existing issue representing this deploy, it will not be moved from the column it is currently in.
+  - The `🚀currently-in-<env>` label will not be removed from the issue that currently has it (if there is one) and it will not put that label on the issue representing this deploy.
```

Comment:
Do you think it would be more clear to bold the not's, or potentially reword it so it says something like `The 🚀currently-in-<env> label will remain on the issue that had it before the workflow ran.`?

Reply:
I'll reword it.